### ISSUE TITLE
Bug when running python code

### ISSUE THREAD

**Issue by xiligey** [twibiral/obsidian-execute-code#157]
Plugin version: 1.2.0

Python version: 3.9.12

Plugin settings:
- Embed Python Plots: yes
- Python path: python
- Python arguments: 
- Run Python blocks in Notebook Mode: yes
- Inject Python code: 

System: Win11

Python code 

```python
import tensorflow as tf
from tensorflow.keras.layers import Dense, Flatten, Conv2D
from tensorflow.keras import Model


class MyModel(Model):

	def __init__(self):
		super().__init__()
		self.conv1 = Conv2D(32, 3, activation='relu')
		self.flatten = Flatten()
		self.d1 = Dense(128, activation='relu')
		self.d2 = Dense(10, activation='softmax')

	def call(self, inputs, training=None, mask=None):
		x = self.conv1(inputs)
		x = self.flatten(x)
		x = self.d1(x)
		return self.d2(x)

	def get_config(self):
		pass

conv1 = Conv2D(32, 3, activation='relu')
print(conv1)
model = MyModel()
print(model)
```

result:
```
name 'Conv2D' is not defined
<keras.layers.convolutional.conv2d.Conv2D object at 0x0000017B16FBFBB0>
```

Questions:
1. why it shows "name Conv2D is not defined"
2. The code `print(model)` seems not work.

Can anyone tell me what's wrong?

**Comment by chlohal** [comment]
Hi! This is a known issue (see #146 #154) which is fixed in PR #147. To hotfix this without waiting for the next release, you can make the following changes:

On line 12054 and 12059 of `<vault folder>/.obsidian/plugins/execute-code/main.js`, remove "`, ${localsName}`"

```diff
          ${printName}(eval(
              compile(${JSON.stringify(code.replace(/\r\n/g, "\n") + "\n")}, "<code block>", "eval"),
-             ${globalsName}, ${localsName}
+             ${globalsName}
          ))
      except SyntaxError:
          exec(
              compile(${JSON.stringify(code.replace(/\r\n/g, "\n") + "\n")}, "<code block>", "exec"),
-             ${globalsName}, ${localsName}
+             ${globalsName}
          )
```

**Comment by twibiral** [comment]
Fixed in release [1.3.0](https://github.com/twibiral/obsidian-execute-code/releases/tag/1.3.0)
